### PR TITLE
CLI - Fix version retrieval.

### LINKED ISSUE #56: inicheck cli version
The `setuptools_scm` version info isn't working for a pip installed version of `inicheck`. Follow what was done for `smrf` which uses `pkg_resources` to load the version number into the `__init__.py`.

### PR DESCRIPTION
**Pull request by jomey**
Get version from metadata using `importlib.metadata`, which is part of
the standard libraries.

Fixes #56

### CODE REVIEW

File: inicheck/cli.py [hunk 1]
```diff
@@ -13,7 +12,18 @@
...
 def current_version():
-    return get_version(root='..', relative_to=__file__)
+    if sys.version_info >= (3, 8):
+        from importlib.metadata import version, PackageNotFoundError
+        try:
+            return version('inicheck')
+        except PackageNotFoundError:
+            'unknown'
+    else:
+        from pkg_resources import get_distribution, DistributionNotFound
```
Comment (scotthavens) [inline inicheck/cli.py:22]: To keep things simpler, I think you can just use pkg_resources. This is what I did in SMRF to get around the python versions that don't have importlib. But as it is now works as well and will be good to see how it works during a pip install.

Comment (jomey) [reply inicheck/cli.py:22]: Good point. Didn't check thoroughly enough that `importlib` was a recent 3.8 introduction. At the very least a good reference for 3.8+ version support.

**Comment by jomey** [comment]
Don't think I have a chance to pass the coverall check. Can you merge this?

**Comment by scotthavens** [comment]
> Don't think I have a chance to pass the coverall check. Can you merge this?

I've updated the settings for coverage so next time this should go throught...

**Pull request merged**